packages/cli/src/services/dynamicNodeParameters.service.ts

```diff
@@ -107,6 +107,7 @@ export class DynamicNodeParametersService {
 			tempNode,
 			{ node, source: null, data: {} },
 			NodeExecuteFunctions,
+			new AbortController().signal,
```

Comment:
For understanding - how are we going to abort this if we don't keep a reference to the controller? Same in `CredentialsHelper`. Or is this only to satisfy the signature? (If the latter, maybe this should be optional?)

Reply:
reverted